Documentation/VMR-re-bootstrapping.md

```diff
@@ -41,8 +41,8 @@ dependency on a new toolset feature until after that feature has been released.
     1. Upload the SDKs to https://dotnetcli.blob.core.windows.net/source-built-artifacts/sdks/
     1. Upload the private source-built artifacts archives to https://dotnetcli.blob.core.windows.net/source-built-artifacts/assets/
 1. Update .NET SDK
-    1. Find the [dotnet-installer-official-ci](https://dev.azure.com/dnceng/internal/_build?definitionId=286)
-    build that best matches the dotnet-source-build. The following is the suggested
+    1. Find the [dotnet-sdk-official-ci](https://dev.azure.com/dnceng/internal/_build?definitionId=140) ([dotnet-installer-official-ci](https://dev.azure.com/dnceng/internal/_build?definitionId=286) for .NET 8) build
```

Comment:
We would never use these instructions to re-bootstrap .NET 8 so I think that link could be dropped.

Reply:
Fixed with https://github.com/dotnet/source-build/pull/4450/commits/08263b19b861918f47eb85932b733010e29a141c